xla/service/gpu/gpu_compiler.cc

```diff
@@ -658,10 +659,11 @@ absl::Status RunSPMDPasses(
                 DefaultAutoShardingOptionFromModuleConfig(hlo_module->config()),
                 alias_info);
           }
-        });
+        }
```

Comment:
You're missing a comma
```suggestion
        },
```